apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/Footer/HostSidePanel/HostSidePanel.tsx

```diff
@@ -34,11 +38,36 @@ export const GET_ALL_TEAM_HOSTS = gql`
 export function HostSidePanel(): ReactElement {
   const { t } = useTranslation('apps-journeys-admin')
   const { journey } = useJourney()
-  const { data, refetch } = useQuery(GET_ALL_TEAM_HOSTS, {
+
+  // Get all team members of journey team, check if user in team
+  // TODO: Replace with CASL authorisation check
+  const { loadUser, data: authUser } = useCurrentUser()
+  useEffect(() => {
+    void loadUser()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+  const team = journey?.team ?? undefined
+  const { data } = useUserTeamsAndInvitesQuery(
+    team != null
+      ? {
+          teamId: team.id,
+          where: { role: [UserTeamRole.manager, UserTeamRole.member] }
+        }
+      : undefined
+  )
+  const userInTeam =
+    data == null || data.userTeams.length === 0 || team == null
+      ? false
+      : data.userTeams.filter(
```

Comment:
I think this should be find since you only need to find the first match?
```suggestion
      : data.userTeams.find(
```